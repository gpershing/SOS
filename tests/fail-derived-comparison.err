Fatal error: exception Failure("Can only equate structs of matching type")
