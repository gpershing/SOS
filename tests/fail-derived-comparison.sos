x: bool = {1,2} == {3,4,5}
