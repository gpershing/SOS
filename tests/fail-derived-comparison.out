Can only equate structs of matching type
